### ISSUE TITLE
[22.02.21] Deberta Model Fine Tuning Results

### ISSUE THREAD

**Issue by Kingthegarden** [medal-contender/nbme-score-clinical-patient-notes#7]
### 1. fc-layer
 - nn.Sequential & dropout(output) : .859 -> 성능향상 X
   - dropout 2번 적용하여 학습이 어려워진 것으로 예상
 - nn.Sequential & output : .864 -> 성능향상 O
### 2. scheduler 변경
 - 기존 Cosine -> CosineAnnealingWarmRestarts 로 변경 : .861 -> 성능향상 X
 ### 3. weight 초기화 방식 변경
 - torch.nn.init.xavier_uniform_를 활용한 Weight Initialization : .866 -> 성능향상 O

#### 해당 내용을 추가하여 pr 보내도록 하겠습니다.
